cmd/vm_show.go

```diff
@@ -89,6 +90,7 @@ func showVM(name string) (outputter, error) {
 		SSHKey:             vm.KeyPair,
 		SecurityGroups:     make([]string, len(vm.SecurityGroup)),
 		AntiAffinityGroups: make([]string, len(vm.AffinityGroup)),
+		PrivateNetworks:    make([]string, 0, len(vm.Nic)-1),
```

Comment:
No need to mess with slice capacity here, either create an empty slice with a 0 length and use `append()` to add elements to it, or create it with final length (`make([]string, len(vm.Nic)-1))` and assign elements using the `for i, nic := range vm.Nic { ... }` position index (`out.PrivateNetworks[i] = vm.Nic[i]`).

Reply:
Why not